### PR TITLE
[Concept Entry] Sklearn multilabel-classification

### LINKED ISSUE #4967: [Concept Entry] Sklearn multilabel-classification
### Reason/inspiration (optional)

We would like a new entry on the `Multilabel Classification` concept under Sklearn. The entry should go in a new file under `docs/content/sklearn/concepts/multilabel-classification/multilabel-classification.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept (if applicable)
- An `Example` section that provides an example demonstrating the concept in use (if applicable)
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use (if applicable)

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by SaviDahegaonkar**
### Description
 Created a new entry on the Multilabel Classification concept under Sklearn.

### Issue Solved
Closes #4967  

### Type of Change

- Adding a new entry

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/sklearn/concepts/multilabel-classification/multilabel-classification.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+Title: 'Multilabel Classification'
+Description: 'Multilabel classification assigns multiple labels to a single instance, using scikit-learn tools and models.'
```
Comment (mamtawardhani) [inline content/sklearn/concepts/multilabel-classification/multilabel-classification.md]:
```suggestion
Description: 'Multilabel classification is a machine learning task where each instance can be assigned multiple labels or categories simultaneously.'
```

File: content/sklearn/concepts/multilabel-classification/multilabel-classification.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  - 'Natural Language Processing'
+  - 'Scikit-learn'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/intermediate-machine-learning-skill-path'
+---
+
+In Sklearn, **Multilabel classification** assigns multiple labels to a single instance using scikit-learn tools and models. This method differs from traditional classification, where each instance belongs to one class, and this technique predicts multiple outputs simultaneously.
+
+Scikit-learn offers tools like `OneVsRestClassifier`, `ClassifierChain`, and `MultiOutputClassifier` to facilitate multilabel classification and enable efficient model training and evaluation.
```
Comment (mamtawardhani) [inline content/sklearn/concepts/multilabel-classification/multilabel-classification.md]:
```suggestion
In sklearn, **Multilabel classification** assigns multiple labels to a single instance, allowing models to predict multiple outputs simultaneously. This method differs from traditional classification, where each instance belongs to only one class.

Scikit-learn offers tools like `OneVsRestClassifier`, `ClassifierChain`, and `MultiOutputClassifier` to handle multilabel classification and enable efficient model training and evaluation.
```

File: content/sklearn/concepts/multilabel-classification/multilabel-classification.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+multi_label_model = MultiOutputClassifier(base_model)
+
+# Step 3: Train the model using the training dataset
+multi_label_model.fit(X_train, y_train)
+
+# Step 4: Make predictions on the test dataset
+predicted_labels = multi_label_model.predict(X_test)
+
+# Step 5: Evaluate predictions or use the results
+print(predicted_labels)
```
Comment (mamtawardhani) [inline content/sklearn/concepts/multilabel-classification/multilabel-classification.md]:
```suggestion
from sklearn.multioutput import MultiOutputClassifier
from sklearn.ensemble import RandomForestClassifier
from sklearn.model_selection import train_test_split

# Step 1: Initialize the base classifier
base_model = RandomForestClassifier(random_state=42)

# Step 2: Create a MultiOutputClassifier wrapper for multilabel classification
multi_label_model = MultiOutputClassifier(base_model)

# Step 3: Train the model using the training dataset
multi_label_model.fit(X_train, y_train)

# Step 4: Make predictions on the test dataset
predicted_labels = multi_label_model.predict(X_test)

# Step 5: Evaluate predictions or use the results
print(predicted_labels)
```

File: content/sklearn/concepts/multilabel-classification/multilabel-classification.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+```
+
+- `RandomForestClassifier`: Used as the base estimator, and can be replaced with any scikit-learn classifier.
+- `MultiOutputClassifier`: Wraps the base model to handle multiple output labels.
+- `Fit`: Method used to train the model on training data.
+- `predict()`: Method makes predicitions on the test data.
```
Comment (mamtawardhani) [inline content/sklearn/concepts/multilabel-classification/multilabel-classification.md]:
```suggestion
- `RandomForestClassifier`: The base classifier for multilabel classification.
- `MultiOutputClassifier`: A wrapper to extend the base classifier for multilabel tasks.
- `Training and testing`: The model is trained with `fit()` and predictions are made using `predict()`.
```

File: content/sklearn/concepts/multilabel-classification/multilabel-classification.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+# Train the model
+model.fit(X, y)
+
+# Predict labels for new data
+predictions = model.predict(X[:5])
+
+# Display predictions
+print("Predicted Labels for First 5 Samples:")
+print(predictions)
```
Comment (mamtawardhani) [inline content/sklearn/concepts/multilabel-classification/multilabel-classification.md]:
```suggestion
from sklearn.datasets import make_multilabel_classification
from sklearn.ensemble import RandomForestClassifier
from sklearn.multioutput import MultiOutputClassifier
from sklearn.metrics import classification_report

# Generate synthetic multilabel data
X, y = make_multilabel_classification(n_samples=100, n_features=10, n_classes=3, n_labels=2, random_state=42)

# Initialize a base classifier
base_classifier = RandomForestClassifier()

# Wrap the base classifier for multilabel classification
model = MultiOutputClassifier(base_classifier)

# Train the model
model.fit(X, y)

# Predict labels for new data
predictions = model.predict(X[:5])

# Display predictions
print("Predicted Labels for First 5 Samples:")
print(predictions)
```

File: content/sklearn/concepts/multilabel-classification/multilabel-classification.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+The code results the following output:
+
+```shell
+Predicted Labels for First 5 Samples:
+[[1 1 1]
+ [1 1 0]
+ [1 1 1]
+ [1 1 0]
+ [0 1 0]]
```
Comment (mamtawardhani) [inline content/sklearn/concepts/multilabel-classification/multilabel-classification.md]:
```suggestion
Predicted Labels for First 5 Samples:
[[1 1 0]
 [1 1 0]
 [0 0 1]
 [1 1 1]
 [0 1 0]]
```

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @SaviDahegaonkar  😄 

The entry looks good to be merged! 🚀
